johnsonandjohnson/Bodiless-JS
# fix: project lock file update

## Test Instructions
1. Checkout the branch
2. Run `npm run setup`
3. Check git statue
4. Expect: no lock file changes
 



## Related Issues
